src/components/MovieDetails/MovieDetails.tsx

```diff
@@ -0,0 +1,36 @@
+import { MovieDetailed } from '../../types/movie-detailed';
+import {
+  MovieDetailsContainer,
+  ImageContainer,
+  InfoContainer,
+  InfoHeader,
+  Rating,
+  Genre,
+  AdditionalInfo,
+  Description,
+} from './MovieDetails.styled';
+
+interface MovieDetailsProps {
+  movie: MovieDetailed;
+}
+
+export function MovieDetails({ movie }: MovieDetailsProps) {
+  return (
+    <MovieDetailsContainer>
+      <ImageContainer>
+        <img alt={movie.name} src={movie.imageUrl}></img>
```

Comment:
In such cases we can use self closing way.
`<img alt={movie.name} src={movie.imageUrl} />`